sp041_20191205T1757.cdl

```diff
@@ -158,10 +158,19 @@ variables:
 		DEPLOYMENT_LONGITUDE:valid_min = "-180" ;
 	string PLATFORM_TYPE ;
 		PLATFORM_TYPE:long_name = "type of glider" ;
-		PLATFORM_TYPE:platform_type_vocabulary = "TBD";
+		PLATFORM_TYPE:platform_type_vocabulary = "TBD" ;
 	string PLATFORM_MODEL ;
-		PLATFORM_MODEL:long_name = "model of the glider" ;
-		PLATFORM_MODEL:platform_model_vocabulary = "TBD";
+		PLATFORM_MODEL:long_name = "Model of the glider" ;
+		PLATFORM_MODEL:platform_model_vocabulary = "TBD" ;
+	string PLATFORM_MAKER ;
+		PLATFORM_MAKER:long_name = "Glider manufacturer" ;
+		PLATFORM_MAKER:platform_maker_vocabulary = "TBD" ;
+	string PLATFORM_SERIAL_NUMBER ;
+		PLATFORM_SERIAL_NUMBER:long_name = "glider serial number" ;
+	string PLATFORM_CODE ;
+		PLATFORM_CODE:long_name = "nickname of a glider" ;
```

Comment:
"nickname of the glider" for consistency

Reply:
Fixed. Thanks!